docs/samples/calling/app.js

```diff
@@ -161,6 +162,54 @@ function changeEnv() {
   enableProduction.innerHTML = enableProd ? 'In Production' : 'In Integration';
 }
 
+const guestUrl = 'https://webexapis.com/v1/guests/token';
+const guestIssuerAccessToken = '';
+
+// Guest access token via Service App
+async function getGuestAccessToken() {
+  const response = await fetch(guestUrl, {
+    method: 'post',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${guestIssuerAccessToken}`,
+    },
+    body: JSON.stringify({
+      subject: 'Guest token for Webex Calling SDK Sample App',
+      displayName: 'Calling Guest User',
+    }),
+  });
+
+  const data = await response.json();
+
+  return data.accessToken;
+}
+
+async function handleServiceSelect(e) {
+  const value = e.target.value;
+
+  if(value === 'guestCalling') {
+    // Show guest inputs and hide access token input
+    jwtTokenForDestElm.classList.remove('hidden');
+    saveElm.disabled = true;
+
+    try {
+      const guestAccessToken = await getGuestAccessToken();
```

Comment:
Maybe we should add a button to fetch the token which is visible based on the dropdown selection instead of populating token directly with the dropdown selection

Reply:
Done